backend/app.ts

```diff
@@ -2,44 +2,59 @@ import * as dotenv from 'dotenv';
 import express from 'express';
 import cors from 'cors';
 import morgan from 'morgan';
-import session from 'express-session';
 import passport from 'passport';
 import googleOAuthRouter from './routes/googleOAuth';
 import testRouter from './routes/testRoute'
 import prisma from './utils/prismaClient';
 import signUpAuth from './routes/signUpAuth';
 import googleStrategy from 'passport-google-oauth20';
+import expressSession from 'express-session';
+import { PrismaSessionStore } from '@quixo3/prisma-session-store';
+import { PrismaClient } from '@prisma/client';
+import { Strategy as LocalStrategy } from 'passport-local';
+import crypto from 'crypto';
+import { secretcode, googleClientID, googleClientSecret, facebookAppSecret, facebookClientID, facebookCallBackURL, googleCallBackURL} from './utils/config'
 import facebookStrategy from 'passport-facebook';
 import facebookOAuthRouter from './routes/facebookOAuth';
 import loginAuth from './routes/loginAuth'
-import { Strategy as LocalStrategy } from 'passport-local';
-import crypto from 'crypto';
-import { secretcode, googleClientID, googleClientSecret, facebookAppSecret, facebookClientID } from './utils/config'
-
+dotenv.config();
```

Comment:
I vaguely remember that we need dotenv.config() to be ran as like one of the first few lines. Should this be higher up? (might not have problems now)